meta-balena-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -16,21 +16,100 @@ run_current_hooks_and_recover () {
 	exit 1
 }
 
+# Help function
+hostapp_update_help() {
+    cat << EOF
+Process host OS update.
+$0 <OPTION>
+
+Options:
+  -h, --help
+        Display this help and exit.
+
+  -f <FILE>, --file <FILE>
+        Update host OS from local docker image file.
+
+  -i <IMAGE>, --image <IMAGE>
+	Update host OS from remote docker image.
+
+  --tmp-dir-symlink <DIR>
+	Path to directory that would be symlinked to be used as balena tmp dir
+	(so that the initial untar of the offline hostos image won't fill up
+	the /mnt/sysroot/inactive partition).
+
+  -r, --reboot
+        Reboot after update.
+
+  -n, --no-hooks
+        Don't process hooks.
+
+  -x, --no-rollback-hooks
+        Don't process rollback hooks.
+EOF
+}
+
```

Comment:
Some background on the `hostapp-update` script is that it's considered an internal API that will change in the future so it is not a documented or supported way to perform a hostOS update. That's mainly why it did not show a help, as doing so is one little step forward to users scripting actions that will break in the future. That doesn't mean we can't ad one, just wanted to add some context.
It would simplify the PR if it did not add the help.